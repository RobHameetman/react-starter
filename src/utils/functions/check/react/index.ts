export * from './isChangeEvent';
export * from './isDragEvent';
export * from './isEventHandler';
export * from './isFocusEvent';
export * from './isFocusEventHandler';
export * from './isKeyboardEvent';
export * from './isMouseEvent';
export * from './isMutableRefObject';
export * from './isPointerEvent';
export * from './isReactElement';
export * from './isScrollEvent';
export * from './isSyntheticEvent';
export * from './isUIEvent';
export * from './isUIEventHandler';
