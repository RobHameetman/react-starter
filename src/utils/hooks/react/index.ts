export * from './useControlProp';
export * from './useEnterEvents';
export * from './useEscapeEvents';
export * from './useFocusEvents';
export * from './usePressEvents';
export * from './usePropsWithChildren';
export * from './useSemanticAsProp';
export * from './useSpaceEvents';
export * from './useTabEvents';
