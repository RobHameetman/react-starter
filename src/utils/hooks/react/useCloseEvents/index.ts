export * from './useCloseEvents';
export { default } from './useCloseEvents';
