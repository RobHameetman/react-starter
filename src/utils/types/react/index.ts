export * from './CC';
export * from './CompoundComponent';
export * from './HandlerPropName';
export * from './HandlerPropsOf';
export * from './PropsWithoutHandlers';
