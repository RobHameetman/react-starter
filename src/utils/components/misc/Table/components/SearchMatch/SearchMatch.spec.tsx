import React from 'react';
import { render } from '@testing-library/react';
import { SearchMatch } from '../../../../../../modules';

describe('SearchMatch', (): void => {
	let html: string | undefined;
	let error: Error | null = null;

	beforeEach((): void => {
		try {
			const { container } = render(<SearchMatch />);

			html = container.innerHTML;
		} catch (err) {
			error = err;
		}
	});

	afterEach((): void => {
		html = undefined;
		error = null;
	});

	it('should render', async (): Promise<void> => {
		expect(html).not.toBeNull();
		expect(error).toBeNull();
	});

	it('should not regress', async (): Promise<void> => {
		expect(html).toMatchSnapshot();
	});
});
