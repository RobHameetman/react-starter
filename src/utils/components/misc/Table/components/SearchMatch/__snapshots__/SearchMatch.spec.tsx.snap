// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchMatch should not regress 1`] = `""`;
